### PR TITLE
refresh readme

### PR DESCRIPTION
**Pull request by slsriehl**
## What?
A description about what this pull request implements and its purpose. Try to be detailed and describe any technical details to simplify the job of the reviewer and the individual on production support.

Update the README to better align with supporting documentation.

## Why?
...

## Testing / Proof
...

@bigcommerce/api-client-developers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,75 @@
...
+cd my-bigcommerce-app
+npm install
+```
+
+3. To expose your app server using an HTTP tunnel, install [ngrok](https://www.npmjs.com/package/ngrok#usage) globally, then start the ngrok service: 
+
+> Note: Use port `3000` to expose the default Next.js server.
+
+```shell
+npm install -g ngrok
```
Comment (rtalvarez) [inline README.md:29]: 🍹 alternatively, for `brew` users: `brew install ngrok`

**Review by rtalvarez — APPROVED** [review]
lgtm 👍

**Review by MariaJose — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,34 +1,75 @@
...
+node -v
+npm -v
+```
+
+2. Clone and/or fork the repo and install npm packages:
+
+```shell
+git clone git@github.com:bigcommerce/sample-app-nodejs.git my-bigcommerce-app
+cd my-bigcommerce-app
+npm install
```
Comment (bc-chaz) [inline README.md:21]: 🍹 a part of me wonders if we should migrate from `npm install` to `npm ci` which would install the same packages from the package-lock file - doing so typically has two benefits: 1) it's faster and 2) it minimizes issues with installing a potentially untested version of a package.

**Review by bc-chaz — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (slsriehl) [reply README.md:29]: done

**Review by bc-chaz — APPROVED** [review]

**Pull request merged**